modules/agent-install-sample-config-bonds-vlans.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/preparing-to-install-with-agent-based-installer.adoc
+
+:_content-type: REFERENCE
+[id="agent-install-sample-config-bonds-vlans_{context}"]
+= Example: Bonds and VLAN interface node network configuration
+
+The following `agent-config.yaml` file is an example of a manifest for bond and VLAN interfaces.
+
+[source,yaml]
+----
+  apiVersion: v1alpha1
+  kind: AgentConfig
+  rendezvousIP: 10.10.10.14
+  hosts:
+    - hostname: master0
+      role: master
+      interfaces:
+       - name: enp0s4
+         macAddress: 00:21:50:90:c0:10
+       - name: enp0s5
+         macAddress: 00:21:50:90:c0:20
+      networkConfig:
+        interfaces:
+          - name: bond0.300 <1>
+            type: vlan <2>
+            state: up
+            vlan:
+              base-iface: bond0
+              id: 300
+            ipv4:
+              enabled: true
+              address:
+                - ip: 10.10.10.14
+                  prefix-length: 24
+              dhcp: false
+          - name: bond0 <1>
+            type: bond <3>
+            state: up
+            mac-address: 00:21:50:90:c0:10 <4>
+            ipv4:
+              enabled: false
+            ipv6:
+              enabled: false
+            link-aggregation:
+              mode: active-backup <5>
+              options:
+                miimon: "150" <6>
+              port:
+               - enp0s4
+               - enp0s5
+        dns-resolver: <7>
+          config:
+            server:
+              - 10.10.10.11
+              - 10.10.10.12
+        routes:
+          config:
+            - destination: 0.0.0.0/0
+              next-hop-address: 10.10.10.10 <8>
+              next-hop-interface: bond0.300 <9>
+              table-id: 254
+----
+<1> Name of the interface.
+<2> The type of interface. This example creates a VLAN.
+<3> The type of interface. This example creates a bond.
+<4> The mac address of the interface.
+<5> The `mode` attribute specifies the bonding mode.
+<6> Specifies the MII link monitoring frequency in milliseconds. This example This example inspects the bond link every 150 milliseconds.
```

Comment:
nit, duplicate words - "This example This example"

Reply:
Good catch , thanks @bfournie , will rectify that